WootricSDK/WootricSDK/WTRiPADThankYouButton.m

```diff
@@ -25,14 +25,15 @@
 #import "WTRiPADThankYouButton.h"
 #import "WTRColor.h"
 #import "UIItems.h"
+#import <WootricSDK/WootricSDK-Swift.h>
```

Comment:
Same question as above

Reply:
When we are using swift code in Objective C then we have to import swift header file like above.